course02/project01/gavin/Asteroids/utils.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <cinder/Area.h>
+#include <glm/vec2.hpp>
+
+enum class Tag {
+    None,
+    Player,
+    Laser,
+    Asteroid
+};
```

Comment:
Again this is a compromise to be able to identify the derived type of a given Collidable when checking collisions.

Reply:
All generic in 76db252356e796ace0396de1cb5a68c9790e3d06.